spec/support/sequel.rb

```diff
@@ -19,8 +19,6 @@ class ParentSequelModel < Sequel::Model
 
     one_to_many :child_sequel_models
 
-    strict_param_setting = true
```

Comment:
This was here for a reason.

http://sequel.jeremyevans.net/rdoc/classes/Sequel/Model/ClassMethods.html#attribute-i-strict_param_setting